modules/mongodb/manifests/s3backup/cron.pp

```diff
@@ -5,13 +5,24 @@
 # [*user*]
 #   The user to run the cronjob as.
 #
-class mongodb::s3backup::cron {
+class mongodb::s3backup::cron(
+  $user = 'govuk-backup'
+) {
+
+  include backup::client
+  require mongodb::s3backup::package
+  require mongodb::s3backup::backup
 
   cron { 'mongodb-s3backup':
     command => '/usr/local/bin/mongodb-backup-s3',
-    user    => $mongodb::s3backup::backup::user,
+    user    => $user,
     minute  => '*/15',
-    require => [Class['mongodb::s3backup::package'],Class['mongodb::s3backup::backup']],
+  }
+
+  cron { 'mongodb-s3-night-backup':
+    command => '/usr/local/bin/mongodb-backup-s3',
+    user    => $user,
+    hour    => '0',
```

Comment:
Thinking about it this also clashes with the other cron job because it's running the same command